### PR TITLE
fix: added cache bursting for translations using file hash

### PR DESCRIPTION
**Pull request by mindhash**
This PR handles cache bursting for translations using following approach:
- a script (i18-generate-hash.js) generates hashes for each of the translation files and writes to i18-translation-hash.JSON file 
- React I18 uses the hashes to generate translation file URL. The hash is added to request URL as follows: 
`/alerts.json?h=3432r33fff ` instead of `/alerts.json`
- The hash generator script (i18-generate-hash.js) is added to package.json in "build" and "dev" commands as a pre-req

### CODE REVIEW

**Comment by mindhash** [comment]
@palashgdev the build pipeline for UI is failing, i am working on it

File: frontend/package.json [hunk 1]
```diff
@@ -4,19 +4,20 @@
 	"description": "",
 	"main": "webpack.config.js",
 	"scripts": {
-		"dev": "cross-env NODE_ENV=development webpack serve --progress",
-		"build": "webpack --config=webpack.config.prod.js --progress",
+		"i18n:generate-hash": "node ./i18-generate-hash.js",
+		"dev": "npm run i18n:generate-hash && cross-env NODE_ENV=development webpack serve --progress",
+		"build": "npm run i18n:generate-hash && webpack --config=webpack.config.prod.js --progress",
```
Comment (palashgdev) [inline frontend/package.json:9]: can we use `prebuild` script here ?

**Review by palashgdev — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 1 above)
Comment (mindhash) [reply frontend/package.json:9]: I tried adding prebuild but it doesn't run when you do yarn dev, yarn lint or playwright command.  The hashes will be generated or update when each of these command run, so I think we have to have hash gen as a separate command.

Comment (palashgdev) [reply frontend/package.json:9]: Got it

**Review by palashgdev — APPROVED** [review]

**Pull request merged**